### PR TITLE
Package view individual panels

### PR DESCRIPTION
**Pull request by andrewdlqnguyen**
At this branch, it will include the 'package' branch from last issue (#13 ) .

4 packages:
Game - To run the game
Model - For the datas
Controller - handle actionlisteners
View

On top, the 'view package' will now include multiple classes of different panel on the GUI. Each one handles different things and are separated to differentiate responsibilities.

### CODE REVIEW

**Comment by andrewdlqnguyen** [comment]
Updated Issue #15

File: src/view/GamePanel.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import java.awt.GridLayout;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+public class GamePanel {
+
+	/* Set up for the game's field of play */
+	private static final int ROWS = 5;
+	private static final int COLS = 10;
```
Comment (vikrambombhi) [inline src/view/GamePanel.java]: You should be getting this information from `model/Board`

File: src/view/MainView.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+import javax.swing.WindowConstants;
+
+//import controller.MainController;
+
+public class MainView extends JFrame {
+	JMenuBar menuBar;
+	JMenu menu1, menu2;
```
Comment (vikrambombhi) [inline src/view/MainView.java]: Bad variable names

File: src/view/PlantSelectionPanel.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package view;
+
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+public class PlantSelectionPanel {
+	private static final int MAX_PLANTS = 5;
```
Comment (vikrambombhi) [inline src/view/PlantSelectionPanel.java:10]: Where did you get this number from?

**Review by vikrambombhi — CHANGES_REQUESTED** [review]

File: src/view/GamePanel.java (hunk 1 above)
Comment (andrewdlqnguyen) [reply src/view/GamePanel.java]: true say!

File: src/view/PlantSelectionPanel.java (hunk 3 above)
Comment (andrewdlqnguyen) [reply src/view/PlantSelectionPanel.java:10]: Random set number. Milestone 3 wants various types of plants, so i just chose 5. We can change it to a more general variable and have a method assign the amount of plants we wish to make.

File: src/view/MainView.java (hunk 2 above)
Comment (andrewdlqnguyen) [reply src/view/MainView.java]: okay!

File: src/view/PlantSelectionPanel.java (hunk 3 above)
Comment (vikrambombhi) [reply src/view/PlantSelectionPanel.java:10]: We should probably make the plant types into a `enum` and use that. We can do that in another PR tho. Can u make a issue for that?

**Pull request merged**